### DISCUSSION TITLE
Where to put github org policy for https://goauthentik.io/integrations/sources/github/#checking-for-membership-of-a-github-organisation

### DISCUSSION THREAD

**Discussion by slimshizn** [goauthentik/authentik#3681] [Q&A]
I setup github oauth with authentik just fine, and would like to add org auth to that. Where exactly is this policy supposed to be? I need a little more information than ``` To check if the user is member of an organisation, you can use the following policy on your flows: ``` Thank you.

**Comment by billmetangmo** [comment]
@slimshizn hav eyou found where to put it ?
